packages/core/src/common/props.ts

```diff
@@ -80,6 +80,7 @@ export interface IOptionProps extends IProps {
 
 /** A collection of curated prop keys used across our Components which are not valid HTMLElement props. */
 const INVALID_PROPS = [
+    "active",
```

Comment:
seems reasonable as `active` is a core modifier. we may add it to other components in the future.